public/docs/_examples/toh-5/ts/app/dashboard.component.ts

```diff
@@ -33,7 +33,7 @@ export class DashboardComponent implements OnInit {
 
   ngOnInit(): void {
     this.heroService.getHeroes()
-      .then(heroes => this.heroes = heroes.slice(1, 5));
+      .then((heroes: Hero[]) => this.heroes = heroes.slice(1, 5));
```

Comment:
My editor was giving me type errors. I thought something changed with the linting. I'll leave it as-is